src/helpers/index.js

```diff
@@ -1,5 +1,21 @@
 import { format } from 'date-fns';
 
+/**
+ * Format a UTC date
+ *
+ * @param {Date} startDate UTC date
+ * @param {string} formatStr pattern to be formatted
+ * @returns {string} Formatted date
+ */
+export const formatUTCDate = (date, formatStr = 'MM/dd/yy') => {
+  const [year, month, day] = date.substr(0, 10).split('-');
```

Comment:
Is possible to replace the `substr` for another method? I was checking the reference and I found this:

> This feature is no longer recommended
> Avoid using it, and update existing code if possible
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/substr